worksheet_D/oxo.py

```diff
@@ -0,0 +1,110 @@
+class OxoBoard:
+    def __init__(self):
+        self.board = {(0, 0): 0,(1, 0): 0, (2, 0): 0,
+                      (0, 1): 0, (1, 1): 0, (2, 1): 0,
+                      (0, 2): 0, (1, 2): 0, (2, 2): 0}      # Creates dictionary.
```

Comment:
OK, but might be neater and more expandable to initialise the dictionary using a loop, e.g.
```
self.board = {}
for x in xrange(3):
    for y in xrange(3):
        self.board[x,y] = 0
```